README.md

```diff
@@ -51,18 +51,17 @@ constants.RID = RID; // IMPORTANT: Without this all castToRID will do is convert
-| `toJSON`                                      | N/A                                                                                                   |                                                          |
-| `formatRecord(record: GraphRecord, opt = {})` | `formatRecord(modelName: string, record: GraphRecord, opt = {})`                                      |                                                          |
-
-PropertyDefinition objects also now store the class name under linkedClass as a string rather than a reference to the actual ClassDefinition instance
+| v3                                                       | v4 equivalent                                                                                                          | Notes                                                    |
+| -------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| `ClassModel.properties`                                  | `SchemaDefinition.getProperties(modelName: string)`                                                                    |                                                          |
+| `ClassModel.required`                                    | `SchemaDefinition.requiredProperties(modelName: string)`                                                               |                                                          |
+| `ClassModel.optional`                                    | `SchemaDefinition.optionalProperties(modelName: string)`                                                               |                                                          |
+| `ClassModel.getActiveProperties()`                       | `SchemaDefinition.activeProperties(modelName: string)`                                                                 |                                                          |
+| `ClassModel.inherits`                                    | `SchemaDefinition.ancestors(modelName: string)`                                                                        |                                                          |
+| `ClassModel.subclasses: ClassModel[]`                    | `SchemaDefinition.children(modelName: string): string[]`                                                               |                                                          |
+| `ClassModel.descendantTree(boolean): ClassModel[]`       | `SchemaDefinition.descendants(modelName: string, opt: { excludeAbstract?: boolean, includeSelf?: boolean }): string[]` | must be called with includeSelf=true to match v3 edition |
+| `ClassModel.queryProperties: Property[]`                 | `SchemaDefinition.queryableProperties(modelName: string): Record<string,PropertyDefinition[]>`                         |                                                          |
```

Comment:
this is still incorrect. it was never an array

Reply:
fixed